### PR TITLE
Upgrade instructor admin module

### PR DESCRIPTION
**Pull request by bgins**
Start upgrade of instructor admin modules to Elm 0.19. All changes were kept within  `instructor_admin/ui/*`. In cases where something was imported by another part of the application, the old versions were left in place.

Some renaming and linting cleanup were done along the way, but we should fix the existing compiler errors before going further. It seems that the compiler is only going so far, and changing more might be risky. (Some of the renaming might in this PR might also be off, but renaming was kept to locals only, and these should be easy to clean up later if needed.)

### CODE REVIEW

File: instructor_admin/ui/Text/Translations/TextWord.elm [hunk 1]
```diff
@@ -16,49 +36,40 @@ type TextWord
...
-            , toString (instance text_word)
-            , toString (phrase text_word)
-            , toString (wordKindToGroup (wordKind text_word))
+            [ String.fromInt (id textWord)
+            , String.fromInt (sectionNumber textWord)
+            , String.fromInt (instance textWord)
+            , phrase textWord
+            , toString (wordKindToGroup (wordKind textWord))
             ]
         ++ ")"
```
Comment (andrew-silvernail) [inline instructor_admin/ui/Text/Translations/TextWord.elm:48]: `textWordToString` is safe to delete.  The `toString` here is trying to convert a `Maybe TextGroupDetails` to a string, this was probably #a debugging function left over from initial dev.

File: instructor_admin/ui/Text/Translations/TextWord.elm [hunk 2]
```diff
@@ -141,86 +152,83 @@ new :
     -> WordKind
     -> Endpoints
     -> TextWord
-new id section instance phrase grammemes translations word endpoints =
-    TextWord id section instance phrase grammemes translations word endpoints
+new wordId section inst _ maybeGrammemes maybeTranslations word endpnts =
+    TextWord wordId section inst phrase maybeGrammemes maybeTranslations word endpnts
```
Comment (andrew-silvernail) [inline instructor_admin/ui/Text/Translations/TextWord.elm:157]: Looks like we are throwing away the phrase param here with `_`. `Text.Translations.TextWord.new`  will still need to use a `phrase` param in order to construct a `TextWord`.  Maybe rename `_` here to `phr` so it doesn't conflict with the `phrase` function (not great but works).

File: instructor_admin/ui/Text/Translations/TextWord.elm (hunk 1 above)
Comment (bgins) [reply instructor_admin/ui/Text/Translations/TextWord.elm:48]: Sounds good, deleting it.

File: instructor_admin/ui/Text/Translations/TextWord.elm (hunk 2 above)
Comment (bgins) [reply instructor_admin/ui/Text/Translations/TextWord.elm:157]: Will do.

**Comment by bgins** [comment]
Thanks for the review! The requested changes are in.

**Comment by andrew-silvernail** [comment]
Thank you!

**Pull request merged**